src/types/types.ts

```diff
@@ -510,10 +635,23 @@ export type EXECUTE = 'EXECUTE';
 
 export type EventChange = EventChangeById | EventChangeByExternalId;
 
+export type SequenceChange = SequencePatch & IdEither;
```

Comment:
Why not interface that extends? More consistent with other similar types

Reply:
answered below